kotlin/codegen/src/main/java/com/squareup/moshi/kotlin/codegen/api/AdapterGenerator.kt

```diff
@@ -159,17 +170,17 @@ internal class AdapterGenerator(
 
   private fun TypeSpec.createProguardRule(): ProguardConfig {
     val adapterProperties = propertySpecs
-        .asSequence()
-        .filter { prop ->
-          prop.type.rawType() == JsonAdapter::class.asClassName()
-        }
-        .filter { prop -> prop.annotations.isNotEmpty() }
-        .mapTo(mutableSetOf()) { prop ->
-          QualifierAdapterProperty(
-              name = prop.name,
-              qualifiers = prop.annotations.mapTo(mutableSetOf()) { it.className }
-          )
-        }
+      .asSequence()
```

Comment:
still should be +4 on wrapped lines, right?

Reply:
Ktlint doesn't support continuation indent anymore, and the compose and kotlin people sold me recently on it being a "you get used to it" thing. The kotlin plugin is iffy on its support of it too

Reply:
Yep, you won't notice in a week.